src/index.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+exports.__esModule = true;
+function sayHi() {
+    console.log('hi');
```

Comment:
Hi :)

Reply:
Hi :)

Reply:
Apart from the whimsical friendliness of this function; do we need it?
(also addNumbers and addValues)

Reply:
Apart from the whimsical friendliness of this function; do we need this?

Reply:
no, that whole file should be gone now :)